td-shim/src/bin/td-shim/td/tdx_mailbox.rs

```diff
@@ -18,6 +18,8 @@ use crate::asm::{ap_relocated_func_addr, ap_relocated_func_size};
 // The maximum size of memory used for AP stacks is 30 KB.
 const MAX_WORKING_AP_COUNT: u32 = 15;
```

Comment:
I filed https://github.com/confidential-containers/td-shim/issues/705 for clean up.